configs/registry/repository-update-hosts.yaml

```diff
@@ -43,7 +43,7 @@ spec:
           echo "Done."
       containers:
       - name: init-container-did-the-work
-        image: gcr.io/google_containers/pause-amd64:3.1@sha256:59eec8837a4d942cc19a52b8c09ea75121acc38114a2c68b98983ce9356b8610
+        image: gcr.io/google_containers/pause:3.1@sha256:f78411e19d84a252e53bff71a4407a5686c46983a2c2eeed83929b888179acea
```

Comment:
Can we update the image version as well? I know from some warnings that there is 3.9 already.

Reply:
I checked the google container registry and only found [3.2](https://console.cloud.google.com/gcr/images/google-containers/global/pause), and is seems like going to be deprecated. May I know where should I look for it?

Reply:
I saw this in a warning unrelated to this: registry.k8s.io/pause:3.6. please verify that's correct replacement. As I see, gcr will be completely deprecated in a month. So, for sure we need to change it.